src/main/telemetry/crsf.c

```diff
@@ -267,17 +267,31 @@ void crsfFrameFlightMode(sbuf_t *dst)
     sbufWriteU8(dst, 0);
     sbufWriteU8(dst, CRSF_FRAMETYPE_FLIGHT_MODE);
 
-    // use same logic as OSD, so telemetry displays same flight text as OSD
-    const char *flightMode = "ACRO";
-    if (airmodeIsEnabled()) {
-        flightMode = "AIR";
-    }
-    if (FLIGHT_MODE(FAILSAFE_MODE)) {
-        flightMode = "!FS";
-    } else if (FLIGHT_MODE(ANGLE_MODE)) {
-        flightMode = "STAB";
-    } else if (FLIGHT_MODE(HORIZON_MODE)) {
-        flightMode = "HOR";
+    // just do "OK" for the moment as a placeholder
+    const char *flightMode = "OK";
+    if (ARMING_FLAG(ARMED)) {
+        if (airmodeIsEnabled()) {
+            flightMode = "AIR";
+        } else {
+            flightMode = "ACRO";
+        }
+        if (FLIGHT_MODE(FAILSAFE_MODE)) {
```

Comment:
Writing this as 

```
if (airmodeIsEnabled()) {
    flightMode = "AIR";
else if (FLIGHT_MODE(FAILSAFE_MODE)) {
[...]
} else {
    flightMode = "ACRO";
}
```

will make it a bit more obvious what is going on.

Reply:
@mikeller  Done